client/app/components/routes/operations/form/Operation.tsx

```diff
@@ -12,7 +13,7 @@ async function getNaicsCodes() {
     return await actionHandler(
       "registration/naics_codes",
       "GET",
-      "/dashboard/operations",
```

Comment:
Looks like your prettier CI action is failing because it wants these commas back in there but I agree I don't like them either.

Reply:
Thanks, looks like this is passing now.